<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndGameDialog</class>
 <widget class="QDialog" name="EndGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Game Over! Here are the results:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="moneyLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Money</string>
   </property>
  </widget>
  <widget class="QLabel" name="foodLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Food</string>
   </property>
  </widget>
  <widget class="QLabel" name="woodLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wood</string>
   </property>
  </widget>
  <widget class="QLabel" name="stoneLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stone</string>
   </property>
  </widget>
  <widget class="QLabel" name="oreLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ore</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1StoneVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1WoodVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1FoodVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1MoneyVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1OreVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2FoodVal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2StoneVal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2MoneyVal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2WoodVal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2OreVal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1NameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2NameLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="winnerLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Winner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="p1Score">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>70</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Outline</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="p2Score">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Outline</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
